nrm_django/grants/models.py

```diff
@@ -575,3 +575,15 @@ class Meta:
 
     # TO-DO: custom save?
     # * Create CN
+
+
+class Category(models.Model):
+    cn = models.CharField(primary_key=True, max_length=34)
+    grant = models.ForeignKey(Grant, models.DO_NOTHING, db_column="grant_cn")
+    category_cd = models.CharField(max_length=2)
+    category_desc = models.CharField(max_length=120, blank=True, null=True)
+    last_update = models.DateField()
```

Comment:
auto_now=True?

Reply:
The database table has `DEFAULT CURRENT_TIMESTAMP NOT NULL`. Is this the Django equivalent?

Reply:
Correct. `auto_now` will update the field each time the object is saved. There's also `auto_now_add` which will do it once on object creation.